pages.es/common/difft.md

```diff
@@ -0,0 +1,33 @@
+# difft
+
+> Compara archivos o directorios basándose en la sintaxis del lenguaje de programación.
```

Comment:
```suggestion
> Compara archivos o directorios en base de la sintáxis del lenguaje de programación.
```

Reply:
sintáxis is not right, _src_: https://dle.rae.es/sintaxis

Reply:
My bad, I'll correct it now.